introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi! I am Filipe and I am really enjoying this course :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?